src/components/HistoryPage.vue

```diff
@@ -71,6 +90,28 @@ export default {
             }
             this.videos = [];
         },
+        onChange() {
+            this.setPreference("autoDeleteWatchHistory", this.autoDeleteHistory);
+            this.setPreference("autoDeleteWatchHistoryDelay", this.autoDeleteDelay);
+            this.setPreference("autoDeleteWatchHistoryTimeUnit", this.autoDeleteTimeUnit);
```

Comment:
Can we have this in one variable? (Maybe store it as seconds or milliseconds, and convert it?)

Reply:
Wouldn't this cause some issues UX-wise?
For example if the user sets the delay to 36 hours, we would probably then show `days` as the selection because the time is gt 24 hours. We could also use some modulos, but that would overcomplicate things in my opinion. Would speaks against storing two different variables in order to keep the code easier to read (in my opinion)?